rasa/core/channels/slack.py

```diff
@@ -69,7 +69,7 @@ async def send_image_url(
         image_block = {"type": "image", "image_url": image, "alt_text": image}
 
         await self._post_message(
-            channel=recipient, as_user=True, text=image, blocks=[image_block]
```

Comment:
I actually would suggest removing this `text` function parameter, since it's not a required one in the method's signature.